games/packwiz/Portfile

```
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup github.com/packwiz/packwiz 7b4be47578151c36e784306b36d251ec2590e50c
```

Nitpick, but can we align things here?

```suggestion
go.setup            github.com/packwiz/packwiz 7b4be47578151c36e784306b36d251ec2590e50c
go.offline_build    no
```

---

The best kind of 'picks.